src/Scaler.Demo/OrderProcessor/Worker.cs

```diff
@@ -21,6 +22,10 @@ internal sealed class Worker : BackgroundService
 
         private ChangeFeedProcessor _processor;
 
+        static Meter s_meter = new Meter("OrderProcessor.CFStore", "1.0.0");
+        private static Counter<int> s_CFRecordsReceived = s_meter.CreateCounter<int>("RecordsReceived");
+        private static Counter<int> s_CFProcessorCount = s_meter.CreateCounter<int>("ProcessorCount");
```

Comment:
Can we also add a prometheus exporter to this :

Reply:
using OpenTelemetry;
using OpenTelemetry.Metrics;

var exporter = new PrometheusExporter(new PrometheusExporterOptions());
var meterProvider = Sdk.CreateMeterProviderBuilder()
    .AddSource("OrderProcessor.CFStore")
    .AddExporter(exporter)
    .Build();